src/qililab/chip/node.py

```diff
@@ -21,7 +21,9 @@
 
 @dataclass(kw_only=True)
 class Node(Settings, FactoryElement):
-    """Class representing a node of the chip's graph."""
+    """This class is used to represent a node of the chip's graph.
+    Each node is an element of the chip
+    """
 
     alias: str
```

Comment:
This attributes are missing definitions in the docs.

Reply:
Same with Port class and its attributes, Qubit, Resonator, Coupler and Coil. Also this classes could benefit from better definition of what the class is about.

Reply:
First pass at this done